src/makeCalculator.js

```diff
@@ -4,7 +4,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    add: (num) => result + num,
+    subtract: (num) => result - num,
```

Comment:
[CODE STYLE] - Use arithmetic operators with assignment operator `=`. Instead of `result - num`, use `result -= num` to subtract the value from `result`.